collections/Laud_Misc/MS_Laud_Misc_129.xml

```diff
@@ -49,59 +60,482 @@
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 104r-105v)</locus>
+                        <rubric>Sermo sancti Augustini de sanctorum mart.</rubric>
+                        <incipit>Solemnitas beatissimorum martyrum lectionem nobis reddidit diem</incipit>
+                        <explicit>ut probatum uellit cum martyribus coronare cui est honor et gloria in saecula saeculorum. Amen.</explicit>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 105v-107r)</locus>
+                        <rubric>Omelia in natiuitate et dedicatio ecclesie</rubric>
+                        <incipit>Quotienscumque, fratres karissimi, <sic>alturis</sic> et templi festiuitatem colimus</incipit>
+                        <explicit>et illud serue bone et fidelis intra in gaudium Domini tui.</explicit>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 107r-108r)</locus>
+                        <rubric>Predicatio sancti Pauli apostoli</rubric>
+                        <incipit><quote>Spiritu ambulate et desideria carnis non perficietis ...</quote> Sed haec contra singula uitia uirtutes singulae opponuntur ut ipso </incipit>
+                        <explicit>ubi certa requies exspectatur indesinenter desiderent</explicit>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 108r-109r)</locus>
+                        <rubric>De Dei timore</rubric>
+                        <incipit>Timor Domini sicut dictum est: <quote>initium sapientiae timor Domini</quote>. Per timorem Domini multa bona possumus promereri </incipit>
+                        <explicit>ut illuc conprehendere possit, uitam caelestem quietem sine fine in saecula saeculorum. Amen. </explicit>
+                     </msItem>
                      <textLang mainLang="la">Latin</textLang>
                   </msItem>
+                  
                   <msItem xml:id="MS_Laud_Misc_129-item4" n="4" class="#vitas_patrum">
+                     <locus>(fols. 109v-137v)</locus>
                      <title type="desc" key="work_13047">Lives of Church Fathers</title>
+                     <rubric>De monacho uage mentis ad salutem reducto</rubric>
+                     <incipit>In uno autem monasterii quidam monachus erat, qui adorationem stare non poterat </incipit>
+                     <note>Includes readings on Severus, Martinus, Gregorius, Benedictus, De puerulo liberato, De offerendis, Eulogius, Apollonius, Ammonius, Copres, Mutius, Piamon, Arsenius, Paphnutius, Eulalius. </note>
                      <textLang mainLang="la">Latin</textLang>
                   </msItem>
+                  
+                  <msItem>
+                     <locus>(fols. 138r-149v)</locus>
+                     <title>Theological work</title>
```

Comment:
Perhaps from Isidore of Seville, Liber sententiarum?

Reply:
yep, excerpts it seems, taken from all three books